crates/ruff_python_formatter/resources/test/fixtures/ruff/notebook_docstring.py

```diff
@@ -0,0 +1,4 @@
+"""
```

Comment:
Can we add another string after this with a semicolon which, I think, should be removed?